src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -2379,20 +2379,17 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
             }
         }
 
-        state.dataHVACVarRefFlow->VRF(VRFNum).FuelType = "Electricity";
         if (!lAlphaFieldBlanks(39)) {
             // A39; \field Fuel type, Validate fuel type input
-            bool FuelTypeError(false);
-            UtilityRoutines::ValidateFuelTypeWithAssignResourceTypeNum(
-                cAlphaArgs(39), state.dataHVACVarRefFlow->VRF(VRFNum).FuelType, state.dataHVACVarRefFlow->VRF(VRFNum).FuelTypeNum, FuelTypeError);
-            if (FuelTypeError) {
+            state.dataHVACVarRefFlow->VRF(VRFNum).FuelTypeNum = static_cast<DataGlobalConstants::ResourceType>(
```

Comment:
These variables should just be `fuelType`. A `Type` (i.e., an enumeration) is a number, so `TypeNum` is redundant.